scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer-page {
+  align-items: center;  
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+    
+  .footer-logo {
```

Comment:
It's not necessary nesting this class definition, please move it outside of here